draft-ietf-quic-http.md

```diff
@@ -85,11 +85,11 @@ subsumed by QUIC, and describes how HTTP/2 extensions can be ported to HTTP/3.
 
 Discussion of this draft takes place on the QUIC working group mailing list
 (quic@ietf.org), which is archived at
```

Comment:
Should `(quic@ietf.org)` be changed as well?

Reply:
I considered switching to `[quic@ietf.org](mailto:quic@ietf.org)`, yeah.  I just have an aversion to mailto: links.  But since you asked...